pilot/pkg/networking/plugin/authz/rbac_test.go

```diff
@@ -432,6 +432,61 @@ func TestConvertRbacRulesToFilterConfig(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
Could you left a TODO in rbac_test.go that we need to refactor all the unit tests soon (we  need to do this before we started working on the galley change).
The current test is becoming more and more hard to maintain.